core/src/main/java/discord4j/core/object/entity/channel/Channel.java

```diff
@@ -167,4 +173,76 @@ public static Type of(final int value) {
             }
         }
     }
+
+    /* Represents channels flags : currently only used in forum channels */
+    @Experimental
+    enum Flag {
+        /**
+         * this thread is pinned to the top of its parent GUILD_FORUM channel
```

Comment:
First in mayus and maybe mention the type of channel related (i mean the class)?